skycatalogs/skyCatalogs.py

```diff
@@ -716,6 +813,20 @@ def get_object_iterator_by_hp(self, hp, obj_type_set=None,
         '''
         pass
 
+    def get_object_by_mjd_interval(self, min_mjd, max_mjd,
```

Comment:
There is no body for this function?  Is there a use case for this function?  If not, I think it would be better to omit it.

Reply:
yeah.  I thought I had deleted it but apparently not.